src/App/Containers/Layout/header.jsx

```diff
@@ -5,19 +5,86 @@ import { withRouter }  from 'react-router';
 import { formatMoney } from '_common/base/currency_base';
 import { urlFor }      from '_common/url';
 import Button          from 'App/Components/Form/button.jsx';
+import UILoader        from 'App/Components/Elements/ui-loader.jsx';
 import {
-    AccountInfo,
     LoginButton,
     MenuLinks,
     SignupButton,
     // ToggleCashier,
-    ToggleMenuDrawer,
     UpgradeButton }    from 'App/Components/Layout/Header';
 import header_links    from 'App/Constants/header-links';
 import { localize }    from 'App/i18n';
 import routes          from 'Constants/routes';
 import { connect }     from 'Stores/connect';
 
+const ToggleMenuDrawer = React.lazy(() => import(/* webpackChunkName: "toggle-menu-drawer" */ 'App/Components/Layout/Header/toggle-menu-drawer.jsx'));
+
+const LazyAccountInfo = ({
```

Comment:
Does it have to be called `LazyAccountInfo`?

Reply:
Not at all, didn't come up with a better name, because this just lazy loads the account info.

Reply:
`AccountInfo` only should be fine, IMO, lazy loads is the implementation, but we use it for account info nevertheless

Reply:
I have used `AccountActions` for the wrapper, seems more correct to me since it contains `LoginButton` and `SignUp` too, if the user is not logged in.